src/main/kotlin/com/vanniktech/android/apk/size/ApkSizeTools.kt

```diff
@@ -0,0 +1,19 @@
+package com.vanniktech.android.apk.size
+
+import java.text.DecimalFormat
+import java.text.DecimalFormatSymbols
+import java.util.Locale
+
+object ApkSizeTools {
+  @JvmStatic fun convertBytesToMegaBytes(bytes: Long): String {
```

Comment:
also I don't think we need the `@JvmStatic here`

Reply:
That was left over from my single file conversion. Will remove.

Reply:
I left `@JvmStatic` for the Spock tests(written in Groovy).

Reply:
Fixed.